src/ResourceManager/Sql/Commands.Sql/help/New-AzureRmSqlDatabase.md

```diff
@@ -285,24 +301,8 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -SampleName
-The name of the sample schema to apply when creating this database.
-
-```yaml
-Type: String
-Parameter Sets: (All)
-Aliases: 
-
-Required: False
-Position: Named
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
 ### CommonParameters
-
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment:
@akromm I agree, it would make it much nicer to view in markdown. I can also talk to the `platyPS` team about how we can prevent this from getting overridden in the future, since it seems to be a hard-coded string that they use for when a cmdlet uses the above set of verbs.